src/AlgebraicGeometry/ToricVarieties/ToricDivisors/properties.jl

```diff
@@ -153,7 +153,10 @@ julia> is_very_ample(td)
 false
 ```
 """
-@attr Bool is_very_ample(td::ToricDivisor) = pm_object(td).VERY_AMPLE
+@attr Bool function is_very_ample(td::ToricDivisor)
+  @req is_complete(toric_variety(td)) "Very ampleness test is only implemented for complete toric varieties. ([Def 6.1.9 CLS11])"
+  return pm_object(td).VERY_AMPLE
```

Comment:
@paemurru, can you please add the type assertion `::Bool`, as suggested by @benlorenz. Then this can be merged.

Reply:
Hmm, I thought I committed the change of @benlorenz , but apparently not.